packages/terra-application-utility/src/utility/_UtilityMenu.jsx

```diff
@@ -78,7 +78,7 @@ class UtilityMenu extends React.Component {
       map: processMenuItems(props.menuItems),
       currentKey: props.initialSelectedKey,
       focusIndex: -1,
-      previousKeyStack: [],
+      previousKeyStack: [], // eslint-disable-line react/no-unused-state
```

Comment:
This lint error seems erroneous, looking through this file we use `previousKeyStack` in a few places. I'm wondering if this is a case that the eslint-react-plugin doesn't account for correctly and is throwing a warning for it.

Reply:
It's exclusively being referenced in setState function arguments; I could understand it being hidden from from the linter.

Reply:
It seems there may be an update to eslint-plugin-react that will view state used in this way as used instead of unused: https://github.com/yannickcr/eslint-plugin-react/issues/1964